cmdeploy/src/cmdeploy/__init__.py

```diff
@@ -522,6 +522,25 @@ def deploy_chatmail(config_path: Path) -> None:
         packages="postfix",
     )
 
+    # Add our OBS repository for dovecot_no_delay
+    files.put(
+        name = "Add Deltachat OBS GPG key to apt keyring",
+        src = importlib.resources.files(__package__).joinpath("obs-home-deltachat.gpg"),
+        dest = "/etc/apt/keyrings/obs-home-deltachat.gpg",
+        user="root",
+        group="root",
+        mode="644",
```

Comment:
Is this necessary? depends on root's umask, I guess. But doesn't hurt.

Reply:
well, I just copied it from the code below, for consistency :D don't know if it was necessary there

Reply:
Yeah, then let's just merge it and try out in a separate PR if we can leave these out.